plugin-build/src/test/kotlin/io/sentry/android/gradle/SentryKotlinCompilerPluginTest.kt

```diff
@@ -0,0 +1,93 @@
+package io.sentry.android.gradle
+
+import java.io.File
+import kotlin.test.assertTrue
+import org.junit.Test
+
+class SentryKotlinCompilerPluginTest :
+    BaseSentryPluginTest(androidGradlePluginVersion = "7.4.0", gradleVersion = "8.0") {
+
+    override val additionalRootProjectConfig: String
+        get() = """
+
+        """.trimIndent()
+
+    override val additionalBuildClasspath: String =
+        """
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.8.20"
+        """.trimIndent()
+
+    @Test
+    fun `does not break for simple compose apps`() {
+        appBuildFile.writeText(
+            // language=Groovy
+            """
+            plugins {
+              id "com.android.application"
+              id "kotlin-android"
+              id "io.sentry.android.gradle"
+              id "io.sentry.sentry-kotlin-compiler-gradle-plugin"
+            }
+
+            android {
+              namespace 'com.example'
+              buildTypes {
+                release {
+                  minifyEnabled = true
+                }
+              }
+              buildFeatures {
+                compose true
+              }
+              composeOptions {
+                kotlinCompilerExtensionVersion = "1.4.4"
+              }
+              kotlinOptions {
+                jvmTarget = "1.8"
+              }
+            }
+            dependencies {
+                implementation "org.jetbrains.kotlin:kotlin-stdlib:1.8.20"
+                implementation "io.sentry:sentry-compose-android:6.18.0-mah-dev-026"
```

Comment:
```suggestion
                implementation "io.sentry:sentry-compose-android:6.18.0"
```
just a reminder to change this after new version is published